### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by wingchunkf**
Created contact page, added navigation bar, and added "recent work" link to navigation bar.
This PR closes issues #12 , #13  and #15.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+
+
+
+
+
+
+ 
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (crobinson42) [inline contact.html]: @wingchunkf Can this be included in the a file in the repo?

**Comment by crobinson42** [comment]
Please fix:
- [ ] Homepage styles are out of wack
screenshot 2017-07-12 08 46 50

- [ ] Top and side padding on input fields don't match on /contact.html
contact_page

- [ ] Add a sumit/send and reset form button

**Comment by wingchunkf** [comment]
Wrote incorrect commit message for commit a2f284d. Added bootstrap.min.js link to html page, rather than actual file. Actual file has  now been committed and pushed.

**Comment by crobinson42** [comment]
@wingchunkf Is this PR good to merge?

**Comment by wingchunkf** [comment]
Yes it is (: 

Christine 

----- Original Message -----

**Pull request merged**